core/app/components/plugins/EventsFilter.tsx

```
@@ -0,0 +1,229 @@
+}
+
+const STATUS_OPTIONS = [
+  { value: 'processed', label: 'Processed', color: 'text-green-600 dark:text-green-400' },
+  { value: 'failed', label: 'Failed', color: 'text-red-600 dark:text-red-400' },
+  { value: 'pending', label: 'Pending', color: 'text-yellow-600 dark:text-yellow-400' },
+];
+
+/**
+ * Events Filter Component
+ */
+export function EventsFilter({ filters, onChange, loading = false }: EventsFilterProps) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'EventsFilter'. Consider to delete it or rename it.

'EventsFilter' is defined here:


(lint/suspicious/noRedeclare)